### PR TITLE
Fix read file xlsx

### PR DESCRIPTION
**Pull request by hardjunior**
TypeError
Argument 2 passed to PhpOffice\PhpSpreadsheet\Style\ConditionalFormatting\ConditionalFormattingRuleExtension::parseExtDataBarAttributesFromXml() must be an instance of SimpleXMLElement, null given, called in

This is:

```
- [X] a bugfix
- [ ] a new feature
```

Checklist:

- [ ] Changes are covered by unit tests
- [ ] Code style is respected
- [X] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [ ] CHANGELOG.md contains a short summary of the change
- [ ] Documentation is updated as necessary

### CODE REVIEW

File: src/PhpSpreadsheet/Style/ConditionalFormatting/ConditionalFormattingRuleExtension.php [hunk 1]
```diff
@@ -80,9 +80,12 @@ public static function parseExtLstXml($extLstXml)
 
                     $extDataBarObj = new ConditionalDataBarExtension();
                     $extFormattingRuleObj->setDataBarExt($extDataBarObj);
-                    $dataBarXml = $extCfRuleXml->dataBar;
-                    self::parseExtDataBarAttributesFromXml($extDataBarObj, $dataBarXml);
-                    self::parseExtDataBarElementChildrenFromXml($extDataBarObj, $dataBarXml, $ns);
+                   
+                    if ($dataBarXml = $extCfRuleXml->dataBar)
```
Comment (MarkBaker) [inline src/PhpSpreadsheet/Style/ConditionalFormatting/ConditionalFormattingRuleExtension.php:84]: It's not really considered good practise to apply an assignment within an `if` test, especially when it's using a loose coercion for truthiness.

**Review by MarkBaker — CHANGES_REQUESTED** [review]
Is this an issue? If so, can you provide an issue number, and a test case to verify that the issue is resolved by this change

**Review by PowerKiKi — CHANGES_REQUESTED** [review]
Mark comment should be addressed and CI must pass before this can be merged. You can run `composer fix` to fix code style locally.

**Comment by oleibman** [comment]
No update in 2.5 years. Closing.

**Pull request closed without merging**